client/state/posts/schema.js

```diff
@@ -51,3 +51,18 @@ export const itemsSchema = {
 	},
 	additionalProperties: false
 };
+
+export const queriesSchema = {
+	type: 'object',
+	patternProperties: {
+		// Queries are JSON strings, optionally prepended by a site ID. We can
+		// assume that query strings contain no spaces, and only simple values.
+		'^(\\d+:)?\\{("[^"]*":("[^"]*"|\\d+|true|false),?)*\\}$': {
```

Comment:
>Since we don't parse this later I'm not sure if we need to be this strict.

My main concern is that there's a real possibility that we change the syntax of the keys. I suppose I could be a bit more lax about it, e.g. `\{[^\}]*\}` instead of the more complete JSON validation I have here.